include/swift/Basic/Features.def

```diff
@@ -411,6 +411,9 @@ EXPERIMENTAL_FEATURE(WarnUnsafe, true)
 /// Import unsafe C and C++ constructs as @unsafe.
 EXPERIMENTAL_FEATURE(SafeInterop, true)
 
+/// Ignore resilience errors due to C++ types.
+EXPERIMENTAL_FEATURE(AssumeResilientCxxTypes, true)
```

Comment:
I think it is better not to use experimental feature since this is a flag to opt out of the error and not really an entire feature.  Think it would better to have a flag that reads like `allow-cxx-types-in-swift-interface`

Reply:
Personally, I prefer the experimental feature approach because they are easier to add and remove. If you add flags, you have to update the drivers and frontend, and possibly leave them in forever. Experimental features are more self-contained.

Reply:
Sounds good to me.